Form-Controls/index.html

```diff
@@ -1,28 +1,105 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
+    <main>
+      <form>
+        <fieldset class="details1">
+          <legend>choose your favorite T-shirt</legend>
+          <div>
+            <label for="black"
+              ><img
+                class="color_img"
+                alt="Black T-shirt"
+                src="https://static.thcdn.com/productimg/1600/1600/13254031-1024901106882108.jpg"
+              /><br />
+              <input
+                type="radio"
+                id="balck"
+                name="colour"
+                required
+              />Black</label
+            >
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
+            <label for="grey"
+              ><img
+                class="color_img"
+                alt="Grey T-shirt"
```

Comment:
Well done on adding descriptive 'alt' text to your images! Lots of people miss this or not use descriptive names. As we also need to cater this for visually impaired users who will be user screen readers.